doc/development/style_guide.md

```diff
@@ -380,21 +372,14 @@ with a width of 600 pixel but it is clickable in the manual page). If a larger
 image is displayed as shrunk, both **width** and **height** HTML parameters
 (values must be calculated according to the picture size!) should be set:
 
-```html
-<div align="center" style="margin: 10px">
-  <a href="r_viewshed.png">
-    <img
-      src="r_viewshed.png"
-      width="600"
-      height="600"
-      alt="r.viewshed example"
-      border="0"
-    /></a><br/>
-  <i>Figure: Viewshed shown on shaded terrain (observer position in the
-    north-east quadrant with white dot; 5m above ground)</i>
-</div>
+```markdown
+![r.viewshed example](r_viewshed.png)  
+*Figure: Viewshed shown on shaded terrain (observer position in the
+north-east quadrant with white dot; 5m above ground)*
 ```
 
+*Note the 2 spaces after the image._
```

Comment:
Remove the `_`?

![image](https://github.com/user-attachments/assets/34719890-c67c-45b1-8d56-13f2726dde3a)